src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="current-directors">
+
+    <v-card flat>
+      <!-- Header -->
+      <div class="header-container">
+        <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
+        <label class="font-weight-bold pl-2">Current Directors</label>
+      </div>
+
+      <!-- Instructional Text -->
+      <div class="instructional-text pt-10 pb-15 px-4">
+        To change directors, please use the Change feature in the Current Directors list on you business dashboard.
+      </div>
+
+      <div class="list-container px-4">
+        <!-- List Display Section -->
+        <div id="people-roles-list" v-if="currentDirectors.length > 0">
+          <!-- List Headers -->
+          <v-row class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
+            <v-col class="pr-5" v-for="(title, index) in tableHeaders" :key="index">
+              <span class="directors-title">{{ title }}</span>
+            </v-col>
+          </v-row>
+
+          <!-- List Content -->
+          <v-row
+            class="people-roles-content py-3"
+            v-for="(orgPerson, index) in currentDirectors"
+            :key="index"
+            no-gutters
+          >
+            <!-- Name + Badge -->
+            <v-col class="text-truncate pr-5">
+              <!-- provide tooltip to display full name if name is longer than 25 chars -->
+              <v-tooltip top :disabled="formatFullName(orgPerson.officer).length < 25" color="primary">
+                <template v-slot:activator="{ on }">
+                  <span v-on="on" class="director-name">{{ formatFullName(orgPerson.officer).toLowerCase() }}</span>
+                </template>
+                <span class="director-name">{{ formatFullName(orgPerson.officer).toLowerCase() }}</span>
+              </v-tooltip>
+            </v-col>
+
+            <!-- Mailing Address -->
+            <v-col class="pr-5">
+              <base-address class="director-detail" :address="orgPerson.mailingAddress" />
+            </v-col>
+
+            <!-- Delivery Address -->
+            <v-col class="pr-5">
+              <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress)"
+                class="director-detail">Same as Mailing Address
+              </p>
+              <base-address v-else class="director-detail" :address="orgPerson.deliveryAddress"/>
+            </v-col>
+
+            <!-- Appoinment Date -->
+            <v-col class="pr-5">
+              <span class="director-detail">{{ orgPerson.roles[0].appointmentDate }} to Current</span>
```

Comment:
What if the person has multiple roles, each with a different appointment  date?

Reply:
We are only showing directors, I filtered it below to only include their role as director. It's not possible to have multiple director roles is it? Not sure how this would work if that's the case

Reply:
Sounds good. Thanks.